### DISCUSSION TITLE
2.6.2 is available - Easy SSL, Easy Settings and Upload to S3 on the fly

### DISCUSSION THREAD

**Discussion by mekya** [ant-media/Ant-Media-Server#5291] [Announcements]
Hey Guys,
We've released the 2.6.2 and it's available to download on [github for community edition](https://github.com/ant-media/Ant-Media-Server/releases/tag/ams-v2.6.2), on [antmedia.io](https://antmedia.io) for enterprise edition. 

Highlights 
-  #5198: 
  Change all settings on the web panel. No need to update the file in the terminal and restart the server 
-  #5082
   Upload HLS files to the S3 on the fly to have a perfect recording and watching experience through S3 -> CDN
-  #4665
   Enable SSL on the web panel. No need to run the command in the terminal

Here is the changelog 

<h3>Ant-Media-Server</h3>
 <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5284'>5284</a> Escape special characters in running command - <a href=''></a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5280'>5280</a> Improve test stability - <a href=''></a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5271'>5271</a> Provide an easy way to change unlisted application settings on web panel - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5198'>https://github.com/ant-media/Ant-Media-Server/issues/5198</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5247'>5247</a> Optimize thread count and update periodicity - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5029'>https://github.com/ant-media/Ant-Media-Server/issues/5029</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5226'>5226</a> Fix snake vulnerability by updating swagger jersey version to 1.6.11 - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5037#issuecomment-1576556276'>https://github.com/ant-media/Ant-Media-Server/issues/5037#issuecomment-1576556276</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5213'>5213</a> Fix URL address - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5212'>https://github.com/ant-media/Ant-Media-Server/issues/5212</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5190'>5190</a> Release/2.6.1 - <a href=''></a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5182'>5182</a> Add REST method to remove subtrack - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5143'>https://github.com/ant-media/Ant-Media-Server/issues/5143</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5162'>5162</a> Upload HLS files to S3 on the fly - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5082'>https://github.com/ant-media/Ant-Media-Server/issues/5082</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/5135'>5135</a> Enable SSL configuration through web panel in a secure way - <a href='https://github.com/ant-media/Ant-Media-Server/issues/4665
'>https://github.com/ant-media/Ant-Media-Server/issues/4665
</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/4969'>4969</a> Pass metadata from processPublishCommand to isPublishAllowed - </li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/4937'>4937</a> Refactor updateStreamSource method - </li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/4869'>4869</a> High resource usage and server unexpected shutdown webhook - <a href='https://github.com/ant-media/Ant-Media-Server/issues/4739'>https://github.com/ant-media/Ant-Media-Server/issues/4739</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Server/pull/4697'>4697</a> Add resolution parameter to on the fly recording - <a href='https://github.com/ant-media/Ant-Media-Server/issues/4640'>https://github.com/ant-media/Ant-Media-Server/issues/4640</a></li><h3>Ant-Media-Enterprise</h3>
<li>Provide an easy way to change unlisted application settings on web panel- <a href=https://github.com/ant-media/Ant-Media-Server/pull/5271 >https://github.com/ant-media/Ant-Media-Server/pull/5271</a></li>
<li>Add hls upload servlet- <a href=https://github.com/ant-media/Ant-Media-Server/issues/5082 >https://github.com/ant-media/Ant-Media-Server/issues/5082</a></li>
<li>check if on the fly track is audio only- <a href=https://github.com/ant-media/Ant-Media-Server/issues/5093 >https://github.com/ant-media/Ant-Media-Server/issues/5093</a></li>
<li>Fix memory leakage- <a href=https://github.com/ant-media/Ant-Media-Server/issues/5029 >https://github.com/ant-media/Ant-Media-Server/issues/5029</a></li>
<li>Fix self track addition for legacy conference solution- <a href=https://github.com/ant-media/Ant-Media-Server/issues/5217 >https://github.com/ant-media/Ant-Media-Server/issues/5217</a></li>
<li>Update version to 2.6.1- <a href= ></a></li>
<li>Remove subtracks from WebRTCClient if they are removed by REST method- <a href=https://github.com/ant-media/Ant-Media-Server/issues/5143 >https://github.com/ant-media/Ant-Media-Server/issues/5143</a></li>
<li>Handle long write trailer case- <a href=https://github.com/ant-media/Ant-Media-Server/issues/4892 >https://github.com/ant-media/Ant-Media-Server/issues/4892</a></li>
<li>Fix blank player on tile count change error- <a href=https://github.com/ant-media/conference-call-application/issues/44 >https://github.com/ant-media/conference-call-application/issues/44</a></li>
<li>Pass metadata from processPublishCommand to isPublishAllowed and send it- </li>
<li>Add resolution parameter to on the fly record- <a href=https://github.com/ant-media/Ant-Media-Server/issues/4640 >https://github.com/ant-media/Ant-Media-Server/issues/4640</a></li>
<h3>StreamApp</h3>
 <li> <a href='https://github.com/ant-media/StreamApp/pull/384'>384</a> fix video elements not getting removed when leaving room - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5281'>https://github.com/ant-media/Ant-Media-Server/issues/5281</a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/381'>381</a> Implement loglevel - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5174'>https://github.com/ant-media/Ant-Media-Server/issues/5174</a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/380'>380</a> Make notifications more visible - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5227'>https://github.com/ant-media/Ant-Media-Server/issues/5227</a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/378'>378</a> Change type to commonjs and add index.js - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5221'>https://github.com/ant-media/Ant-Media-Server/issues/5221</a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/377'>377</a> Fix enableAudioLevel function not working error - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5218'>https://github.com/ant-media/Ant-Media-Server/issues/5218</a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/376'>376</a> Fix 360 player's mute/unmute button - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5197'>https://github.com/ant-media/Ant-Media-Server/issues/5197</a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/375'>375</a> Check RTCDataChannel state on sendData - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5159'>https://github.com/ant-media/Ant-Media-Server/issues/5159</a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/374'>374</a> Bump socket.io-parser from 4.2.2 to 4.2.4 - <a href=''></a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/373'>373</a> Release/2.6.1 - <a href=''></a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/372'>372</a> Replace alerts with notify - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5170'>https://github.com/ant-media/Ant-Media-Server/issues/5170</a></li> <li> <a href='https://github.com/ant-media/StreamApp/pull/371'>371</a> Upload HLS files to S3 on the fly - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5082'>https://github.com/ant-media/Ant-Media-Server/issues/5082</a></li><h3>Ant-Media-Server-Parent</h3>
 <li> <a href='https://github.com/ant-media/ant-media-server-parent/pull/101'>101</a> Release/2.6.1 - <a href=''></a></li><h3>Ant-Media-Management-Console</h3>
 <li> <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/227'>227</a> Rename DASH Streaming -> DASH & CMAF Streaming - <a href='https://github.com/ant-media/Ant-Media-Server/issues/4743'>https://github.com/ant-media/Ant-Media-Server/issues/4743</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/226'>226</a> Support updating all application settings on web panel - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5198'>https://github.com/ant-media/Ant-Media-Server/issues/5198</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/223'>223</a> Revert jsonwebtoken to 8.5.1 becacuse it has instanceof error - <a href='https://github.com/ant-media/Ant-Media-Server/issues/5240'>https://github.com/ant-media/Ant-Media-Server/issues/5240</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/213'>213</a> Error in first enter to application when activated JWT app filter issue fix - <a href='https://github.com/ant-media/Ant-Media-Server/issues/4217'>https://github.com/ant-media/Ant-Media-Server/issues/4217</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/210'>210</a> SSL through web panel changes and scroll issue fix - <a href='https://github.com/ant-media/Ant-Media-Server/issues/4665
https://github.com/ant-media/Ant-Media-Server/issues/4670'>https://github.com/ant-media/Ant-Media-Server/issues/4665
https://github.com/ant-media/Ant-Media-Server/issues/4670</a></li> <li> <a href='https://github.com/ant-media/Ant-Media-Management-Console/pull/208'>208</a> fix: checkbox placement for enabling low-latency HLS Streaming CMAF - </li>

**Comment by felixmeedt** [comment]
Hi,

in my Downloads there´s no 2.6.2 Enterprise existing.

Also in Azure Marketplace there is no 2.6.2.

When will it appear?

Thanks

**Comment by muratugureminoglu** [comment]
Hi @felixmeedt 

When I checked, I can use version 2.6.2 of Ant Media Server Enterprise from the Azure Marketplace. Which zone are you using?

![Screenshot at 2023-08-07 12-52-30](https://github.com/ant-media/Ant-Media-Server/assets/6337605/15358d46-9146-49b1-be4b-95f9e818a2b6)
